### DISCUSSION TITLE
Show diff from nvim and neotest

### DISCUSSION THREAD

**Discussion by lawrence-laz** [VerifyTests/Verify#847] [Q&A]
I am trying to set up Verify with nvim and [neotest](https://github.com/nvim-neotest/neotest).

The tests run fine, but the diff is not being opened.

![image](https://user-images.githubusercontent.com/8823448/232144170-1069501c-b145-4a8e-8d47-8b7a78d64a36.png)

When I run `dotnet test` from a terminal directly the diff does open.

I wonder what is the difference between them? And how could I go about getting the diff show up when running tests from withing nvim and neotest?

**Comment by SimonCropp** [comment]
can u try debugging into it to see where it gets to?
